actionw/ACTIONW_WPIMULT-02.DATA

```diff
@@ -0,0 +1,1065 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+  
+-- Copyright (C) 2022 Equinor
+                                                                           
+-- The model is the base model for testing the WPIMULT keyword in an ACTIONW block.
+--

```

Comment:
Please include one line here from the ACTIONW.md file to explain how this case differs from the ACTIONW_WPIMULT-01.DATA.